indi-gpsd/gps_driver.cpp

```diff
@@ -279,8 +282,11 @@ IPState GPSD::updateGPS()
             break;
     }
 
-
+#if GPSD_API_MAJOR_VERSION >= 10
+    if (gpsFix->status == STATUS_NO_FIX)
```

Comment:
This change is bogus as the pointer is not initialized anywhere. The status field has indeed moved so the check should be:
if (gpsData->fix.status == STATUS_NO_FIX)